webpack/stripes-duplicate-plugin.js

```diff
@@ -7,25 +7,23 @@ const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack
 // Module names that must not have duplicates
 const duplicatesNotAllowed = [
   'react',
+  'react-dom',
+  'react-intl',
+  'react-router',
+  'react-router-dom',
+  'redux',
+  'stripes',
   'stripes-core',
   'stripes-components',
-  'stripes-smart-components',
-  'stripes-form',
   'stripes-connect',
-];
-
-// Module names that are acceptable to have duplicates
-const duplicatesAllowed = [
+  'stripes-final-form',
+  'stripes-form',
+  'stripes-smart-components',
```

Comment:
Does it makes sense to also add the package about react-intl-safe-html?

Reply:
I don't think that one matters. Do you have reason to believe it does?

Reply:
I don't think the issue there is dupes of that package, but that it _currently_ implies that somebody is expecting an older version of react-intl. It doesn't even bring in react-intl itself so that's not an issue. And future versions could very well coexist peacefully.

Reply:
Answering to Zak's questions, I just had a concern that multiple versions will not coexist peacefully, but it seems like yeah, the root cause is multiple versions of react-intl, which is already addressed.